tests/core.js

```diff
@@ -33,10 +33,10 @@ require([
             util.init({
                 core: {
                     onReady: function () {
-                        var dup;
                         util.addQuestion("Text", "question1");
-                        dup = util.addQuestion("Text", "question2");
-                        dup.p.nodeID = "question1";
+                        util.addQuestion("Text", "question2");
+                        var target = $("[name='property-nodeID']");
+                        target.val('question1').change();
```

Comment:
Are you referring to [this set method](https://github.com/dimagi/Vellum/blob/disallow-duplicate-choice/src/mugs.js#L75)? Should investigate why `dup.p.nodeID = "question1";` does trigger change events, etc. since as far as I know many things depend on that.